### DISCUSSION TITLE
Error message when importing Business Process Catalog to Azure DevOps

### DISCUSSION THREAD

**Discussion by stinemarist** [microsoft/dynamics365patternspractices#200] [Q&A] [answered]
Hello, 

We are trying to import the Business Process Catalog to Azure DevOps, but we can't get past this error message:
![2024-04-17_08-47-17](https://github.com/microsoft/dynamics365patternspractices/assets/164178178/fa44f651-74eb-4892-b868-616acd254c5a)

We have followed the steps on the Tech Talk about this subject and followed the steps on Microsoft Learn. Does anyone have any ideas on what we have done wrong and how we can fix this issue? :)

Thank you in advance

Regard, 
Stine

**Comment by FH-Inway** [comment]
It seems based on the error message that you semicolons as field separator in the csv file. That may happen if you created the csv file from Excel that uses e.g. a German localization. If that is the case, try replacing the semicolons with colons. Otherwise please provide a small sample of one of the csv files.

**Reply by stinemarist** [reply]
Hello, 

Thank you for responding. I will attach the file we have tried to import. 
[CSV test 2.csv](https://github.com/microsoft/dynamics365patternspractices/files/15075209/CSV.test.2.csv)

**Comment by rachel-profitt** [comment, marked as answer]
The file you have shared is semicolon separated instead of comma separated which is what Azure DevOps expects. 

When your save a workbook as a .csv file, Excel separates values with your default List separator. To force it to use a different delimiter, proceed with the following steps:
1. Click **File > Options > Advanced.**
2. Under **Editing options**, clear the **Use system separators** check box.
3. Change the default **Decimal separator**. As this will change the way decimal numbers are displayed in your worksheets, choose a different **Thousands separator** to avoid confusion.
4. Depending on which separator you wish to use, configure the settings in one of the following ways.

To convert Excel file to CSV semicolon delimited, set the default decimal separator to a comma. This will get Excel to use a semicolon for the List separator (CSV delimiter):
- Set Decimal separator to comma (,)
- Set Thousands separator to period (.)
